src/screens/KeyManagement/OCR2KeysCreate.tsx

```diff
@@ -19,6 +19,9 @@ export const OCR2KeysCreate: React.FC<Props> = ({
     string | undefined
   >(undefined)
 
+  const { loading, error, data } = useOCR2KeyFamilyQuery()
```

Comment:
Can we move this all the way to the top?

Reply:
This is the highest it can go otherwise it will break `setState`